chat-plugins/mafia.js

```diff
@@ -919,18 +1148,18 @@ exports.pages = {
 			buf += `<p>If you have roles that have conflicting alignments or base roles, you can use /mafia forcesetroles [comma seperated list of roles] to forcibly set the roles.</p>`;
 			buf += `<p>Please note that you will have to PM all the players their alignment, partners (if any), and other infromation about their role because the server will not provide it.</p>`;
```

Comment:
There's a typo on this line if you want to fix it.
Infromation should be information.

Reply:
Will fix later as I don't have time to update now.